CMakeLists.txt

```diff
@@ -190,13 +197,16 @@ if(UMF_BUILD_LIBUMF_POOL_JEMALLOC)
 endif()
 
 # set UMF_PROXY_LIB_ENABLED
-if(WINDOWS AND NOT (CMAKE_BUILD_TYPE STREQUAL "Release"))
-    # TODO: enable the proxy library in the Debug build on Windows
+if(WINDOWS)
+    # In MSVC builds, there is no way to determine the actual build type during
+    # the CMake configuration step. Therefore, this message is printed in all
+    # MSVC builds.
     message(
         STATUS
-            "Disabling the proxy library, because it is supported only in the Release build on Windows (CMAKE_BUILD_TYPE = ${CMAKE_BUILD_TYPE})"
+            "The proxy library will be built, however it is supported only in the Release build on Windows"
     )
-elseif(UMF_PROXY_LIB_BASED_ON_POOL STREQUAL SCALABLE)
+endif()
+if(UMF_PROXY_LIB_BASED_ON_POOL STREQUAL SCALABLE)
```

Comment:
shouldn't this be still `elseif`?

Reply:
No, because on Windows we build the proxy lib, too. Now on all configurations.

Reply:
ahh, right